### DISCUSSION TITLE
Screenshot styles

### DISCUSSION THREAD

**Discussion by Psycodex2001** [dynobo/normcap#285] [Q&A]
**Can the normcap take a full window and its page screenshot?**

_E.g.:_

![Screenshot 2022-10-10 at 17-01-21 full window screenshot - Google Search](https://user-images.githubusercontent.com/81395301/194943722-6cd1b772-c6a3-4fd5-94c1-d4df8a009864.png)

> But with the following toolbar, tabs, and the other menus included on a unique screenshot:

![image](https://user-images.githubusercontent.com/81395301/194943985-a18b5034-13db-421e-8eb4-ac8d74591165.png)

**And take a window pop-up (Like dialog boxes, context menus, web poop-ups) full screenshot?**

_E.g.:_

![Screen-Shot-2021-01-22-at-2 07 44-PM-1024x582](https://user-images.githubusercontent.com/81395301/194948552-b4c99157-60d3-41fa-8c9c-af223cf8e1bf.png)

> like this, a gmail window pop-up screenshot, but showing its content instead, and maybe showing the toolbar, tabs, and the other menus too!

Anyways, thanks so much for the app creation and support!

**Comment by dynobo** [comment]
Hi @Supersity.

Unfortunately, no, that is not possible right now and most likely won't be in the near future.

In general, the purpose of NormCap is to **extract sections of text from a visible screen**. It is _not_ intended to capture invisible parts of the screen, and it also is _not_ intended to provide images of screen-content (other tools do this much better).

For your specific use-case, I'm also not aware of another tool which could help: There are tools for creating a screenshot of a whole webpage, and there are tools for creating a screenshot from the Browser UI, but I never heard of a tool that does both.

I hope this answers your question and you find a solution to your problem!

**Reply by Psycodex2001** [reply]
Thanks, bro!
Actually, I depend on these types of tools because I need to prove a purchase by a screenshot on my Gmail and until now I have never discovered how, so disgusting... But with ShareX I could take a screenshot of the whole screen, but with merged sequential shots in one photo.

Don't you know any software that can unify all the separated photos in one big one using any algorithm?


Here is a sample of how the photo file is:
![Screenshots Proof sample](https://user-images.githubusercontent.com/81395301/198852973-3b57a87c-9013-45bb-888c-b5117a791256.jpg)

**Comment by dynobo** [comment]
The closest to what I think you need seems to be Chrome extensions like [Nimbus Screenshot](https://chrome.google.com/webstore/detail/nimbus-screenshot-screen/bpconcjcammlapcogcnnelfmaeghhagj) ("selected and scroll"-feature) or maybe [Fireshot](https://chrome.google.com/webstore/detail/take-webpage-screenshots/mcbpblocgmgfnpjjppndjkmgjaogfceg). There are many extensions like those which you can try, but it seems like your use case is very special, so I'm not sure if there really is something useful to you out there. 
(It seems rather odd, that you are required to prove that via_screenshots_, rather than exporting the mail thread to PDF or something. Feels like they _want_ you to do a task, that's very hard to automate).

Good luck!
